BridgeCareApp/BridgeCare/DataAccessLayer/LatLonSection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BridgeCare.ApplicationLog;
+using BridgeCare.Interfaces;
+using BridgeCare.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.DataAccessLayer
+{
+    public class LatLonSection : ILatLonSection
+    {
+        public LatLonSection() { }
+        public IQueryable<LatLonSectionModel> GetLatLon(int NetworkId, BridgeCareContext db)
+        {
+            IQueryable<LatLonSectionModel> rawQueryForData = null;
+
+            // bug in program: Long is null in rolled up DB even when the source DB has valid data, including isnull 
```

Comment:
You can use the [HACK] keyword when handling "weird" issues like this one. You can even link the bug branch in the [HACK] comment. Once the bug is resolved, you can remove the [HACK] portion of the comment.

Reply:
good to know, just changed comment as using isnull will probably stay here regardless of the outcome, null protection for non null-able types in general needs to be accounted for.